tools/scripts/read-output-params.ps1

```diff
@@ -0,0 +1,37 @@
+<#
+ .SYNOPSIS
+    Sets pipeline variables
+
+ .DESCRIPTION
+    The output of the AzureResourceManagerTemplateDeployment task is available via deploymentOutputs variable.
+    This script reads that variable, parses it and sets the resource manager deployment output such as they can
+    be used by other tasks.
+
+ .PARAMETER armOutputString
+    The output of the AzureResourceManagerTemplateDeployment task
+#>
+
+param (
+    [Parameter(Mandatory=$true)]
+    [string]
+    $armOutputString = ''
+)
+
+Write-Output "Retrieved input: $armOutputString"
```

Comment:
Monika, here you print whole `$armOutputString` to console, which also includes secrets passed through it. Please do not output whole content.